promiseFOur.then((user) => { //avoid callback hell
//     console.log(user);
//     return user.username
// }).then((username) => {
//     console.log(username);
// }).catch(function (error) {
//     console.log(error);

// }).finally(() => console.log("Finally the promise is resolved or rejected")
// )